Lib/test/test_bz2.py

```diff
@@ -1001,6 +1005,8 @@ def test_encoding_error_handler(self):
                 as f:
             self.assertEqual(f.read(), "foobar")
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
```

Comment:
@coolreader18 @arihant2math what cause this regression? could this be fixed in future?

Reply:
A flag probably needs to be added to the decompressor state. I believe the issue is the `wt` and `rt` formats. Although I'm not to sure, I suppose looking at the cpython source might help.

Reply:
The reason this needed to be flagged now is because the whole test file wasn't running at all before - bz2 was not included in the `--features=stdlib,threading,...` flag in CI, so the module wasn't even getting compiled or tested at all.

Reply:
Oh, didn't noticed that. Thanks!